farm/modeling/adaptive_model.py

```diff
@@ -40,7 +40,7 @@ def __init__(
         """
         super(AdaptiveModel, self).__init__()
         self.language_model = language_model.to(device)
-        self.prediction_heads = [ph.to(device) for ph in prediction_heads]
+        self.prediction_heads = nn.ModuleList([ph.to(device) for ph in prediction_heads])
```

Comment:
Have you tested training, saving and loading for the examples after introducing this?

Reply:
I have tested training on all tasks.
I haven't tested saving, but this shouldn't impact saving at all.
I loaded a QA model that was saved with the old code and it worked. Haven't tested the other models.